nuget/OpenTelemetry.AutoInstrumentation.nuspec

```diff
@@ -0,0 +1,68 @@
+        <dependency id="OpenTelemetry.Exporter.OpenTelemetryProtocol" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Prometheus" version="1.3.0-rc.2" />
+        <dependency id="OpenTelemetry.Exporter.Zipkin" version="1.3.1" />
+        <dependency id="OpenTelemetry.Extensions.Propagators" version="1.3.1" />
+        <dependency id="OpenTelemetry.Instrumentation.GrpcNetClient" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Http" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Runtime" version="1.0.0" />
+        <dependency id="OpenTelemetry.Instrumentation.SqlClient" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Wcf" version="1.0.0-rc.7" />
+        <dependency id="OpenTelemetry.Shims.OpenTracing" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.AspNetCore" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs" version="1.3.0-rc.2" />
+      </group>
+      <group targetFramework=".NETFramework4.6.2">
+        <dependency id="OpenTelemetry" version="1.3.1" />
+        <dependency id="OpenTelemetry.Api" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Console" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Jaeger" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.OpenTelemetryProtocol" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Prometheus" version="1.3.0-rc.2" />
+        <dependency id="OpenTelemetry.Exporter.Zipkin" version="1.3.1" />
+        <dependency id="OpenTelemetry.Extensions.Propagators" version="1.3.1" />
+        <dependency id="OpenTelemetry.Instrumentation.GrpcNetClient" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Http" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Runtime" version="1.0.0" />
+        <dependency id="OpenTelemetry.Instrumentation.SqlClient" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Wcf" version="1.0.0-rc.7" />
+        <dependency id="OpenTelemetry.Shims.OpenTracing" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.AspNet" version="1.0.0-rc9.5" />
+      </group>
+    </dependencies>
+    <frameworkAssemblies>
+      <frameworkAssembly assemblyName="System.Configuration" targetFramework=".NETFramework4.6.2" />
+      <frameworkAssembly assemblyName="System.Web" targetFramework=".NETFramework4.6.2" />
+      <frameworkAssembly assemblyName="System.ServiceModel" targetFramework=".NETFramework4.6.2" />
+    </frameworkAssemblies>
+  </metadata>
+  <files>
+    <file src="bin-ubuntu-20.04/AdditionalDeps/**/*.*" target="AdditionalDeps" />
+    <file src="bin-ubuntu-20.04/netcoreapp3.1/**/*.*" target="netcoreapp3.1" />
+    <file src="bin-ubuntu-20.04/store/**/*.*" target="store" />
```

Comment:
What if Windows-store!=ubuntu-store!=macos-store!=alpine-store?

Reply:
This is a very good question. Our current design does not allow the different stores for these platforms, so we may not run into this issue. If the store logic changes we might end up with this issue. Currently, runtime store is built based on .NET version and not platform.

Reply:
It could be separate issue, but it will be great to have test if all of them are the same.

Reply:
They should be the same in package level, as AFAIK you cannot publish different packages in the same name.
You would just want to include **every** difference.

Also this comment is addressing the same issue: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1368#issuecomment-1273432658